### ISSUE TITLE
Exception when uploading in ios

### ISSUE THREAD

**Issue by amanthegreatone** [mybigday/react-native-s3#45]
Hi,

getting an exception when trying to upload in ios. The same code works perfectly on android device.

![screen shot 2017-03-04 at 12 07 23 pm](https://cloud.githubusercontent.com/assets/24354452/23576730/93ef3d78-00d3-11e7-8b12-d7d8f808af72.png)

iPhone 6
RN 0.40.0
react-native-s3 0.0.24

**Comment by jhen0409** [comment]
It's valid bucket name `syook-saas-dev/cache`? The bucket name should be DNS-compliant.

**Comment by amanthegreatone** [comment]
yes the bucket is valid. The same code is working on android device perfectly.

**Comment by amanthegreatone** [comment]
can someone help me troubleshoot this?
